sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/models/ece-integration.model.ts

```diff
@@ -0,0 +1,48 @@
+import { DisbursementValue } from "@sims/sims-db";
+
+export const ECE_SENT_TITLE = "CONFIRMATION REQUEST";
+
+/**
+ * Result of a file uploaded to SFTP on ZONE B network.
+ */
+export interface ECEUploadResult {
+  generatedFile: string;
+  uploadedRecords: number;
+}
+
+/**
+ * Represents a single line in a ECE request file.
+ * When implemented in a derived class this
+ * interface allow the object to be represented
+ * as a formatted fixed string.
+ */
+export interface ECERequestFileLine {
+  getFixedFormat(): string;
+}
+
+export interface ECERecord {
+  institutionCode: string;
+  disbursementValues: DisbursementValue[];
+  sin: string;
+  studentLastName: string;
+  studentGivenName: string;
+  birthDate: string;
+  sfasApplicationNumber: string;
```

Comment:
This is not the SFAS application number, right?
PS: I really do not care what it is called in the documentation 😉